.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+    "presets": ["@babel/preset-env", "@babel/preset-react"],
+    "plugins": [["@babel/plugin-proposal-class-properties"]]
+}
```

Comment:
Beth's code requires a few extra parameters and dependencies to work, Make sure you have everything!